docs/lib/graphqlapi/fragments/android/authz/22_none_interceptor.md

```diff
@@ -0,0 +1,41 @@
+You can also register your own request interceptor to intercept the request and perform an action or inject something into your request before it it performed. 
+
+TODO: this file can be deleted, I don't think Android has a general request interceptor pattern
```

Comment:
need to confirm

Reply:
Something was added not too long ago. The docs currently have it listed under https://docs.amplify.aws/lib/graphqlapi/advanced-workflows/q/platform/android#adding-headers-to-outgoing-requests

Reply:
Not sure if it would be better to copy that snippet here or simply link to that section of the docs. Thoughts?

Reply:
oh nice, simply reusing it looks good!